mer-android-chroot

```diff
@@ -226,6 +226,15 @@ umount_spare() {
     umount ${uburoot}/dev/pts || :
     umount ${uburoot}/dev/shm ${uburoot}/run/shm || :
     umount ${uburoot}/var/run/dbus || :
+    umount ${uburoot}/sys || :
+    umount ${uburoot}/dev || :
+    umount ${uburoot}/proc || :
+
+    if [[ $bind_mount_home == yes ]] ; then
+        if mountpoint -q ${uburoot}/home/$user ; then
```

Comment:
In mount part the path can also be something else, should similar way be done here?